cypress/integration/basic.ts

```diff
@@ -1,13 +1,13 @@
-import { BAR_HEIGHT } from '../../webapp/javascript/components/FlameGraph/FlameGraphComponent/index.jsx'
+import { BAR_HEIGHT } from '../../webapp/javascript/components/FlameGraph/FlameGraphComponent';
```

Comment:
Does this cause a problem or is the `/index.jsx` not needed?

Reply:
That's an `eslint` error
```
Unexpected use of file extension "jsx" for "../../webapp/javascript/components/FlameGraph/FlameGraphComponent/index.jsx"  import/extensions
```

We weren't catching it before because our lint rule was only applied to `js/jsx` files (this one is a `ts` file).

But overall it's not needed, AFAIK all tools follow convention that `foo/index.{js,jsx,ts,tsx}` equal to `/foo`